examples/postgrescluster/postgrescluster.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: postgres-operator.crunchydata.com/v1alpha1
+kind: PostgresCluster
+metadata:
+  name: mycluster
+spec:
+  instances:
+    - name: instance1
```

Comment:
I don't see these fields in the CRD, yet.

Reply:
I just had to add _something_ to the spec to get the Kube API to accept the `postgrescluster`, so I just went ahead and added these per the currently proposed CRD design (figure these will be some of the first fields we add).